logstash-core-event-java/src/main/java/com/logstash/Event.java

```diff
@@ -119,7 +121,16 @@ public void setTimestamp(Timestamp t) {
         this.data.put(TIMESTAMP, this.timestamp);
     }
 
+    public Object getFieldForRuby(String reference) {
+        return _getField(reference);
+    }
+
     public Object getField(String reference) {
+        Object val = _getField(reference);
+        return Valuefier.unconvert(val);
+    }
+
+    private Object _getField(String reference) {
```

Comment:
I am not sure I like the underscore naming here `_getField` - it does not convey the specific purpose of this method, not is it documented